api/models/userModel.js

```diff
@@ -13,4 +13,17 @@ const User = db.define('users', {
   }
 }, { timestamps: false })
 
+User.associate = function (models) {
+  User.hasMany(models.CollectionUser, {
+    as: 'user',
```

Comment:
as is alias for collectionuser so it should not say collection as collection is the main table and associated table should have different alias possibly followers or just collectionuser